Documentation/Signoffs/Nav-Microcontroller-Signoff.md

```diff
@@ -0,0 +1,127 @@
+</p>
+
+### Sensor Connections
+
+<p align = "center">
+<img src = "https://github.com/lchapman42/Control-Sensing-Wireless-Charging-Robot/blob/main/Documentation/Images/Nav-Microcontroller/Sensor_Connections.png?raw=true">
+</p>
+
+## Analysis
+
+### Arduino Analysis
+The Arduino Mega 2560 Rev3 has 54 digital pins and 16 analog pins. It also has 15 PWM output pins. The PWM pins will give future teams the option to attach servo or stepper motors to complete their tasks without having to implement another microcontroller. The Mega has 4 sets of Tx/Rx pins for UART communication. In addition to the UART pins, there are SDA and SCL pins for I2C communication. The microcontroller has an additional 4 kb of EEPROM that does not get erased from powering down. 
+
+The input/supply voltage to the Arduino Mega can span from 7 to 12 V from the DC jack and the VIN pin. Powering through the USB has a range of 4.8 to 5.5 V, but since the communication from Mega to Jetson Nano will use the USB for data transmission, this is not an option for powering the microcontroller.
+
+The Arduino serial monitor and serial plotter will aid in validating the voltages received from the sensors. The plotter will also aid in creating software filters. Each of the sensors will be tested with the plotter active so that votlage readings are known for a controlled reading. 
+### Arduino Libraries
+The sensor sets used for the robust robot will utilize manufacturer provided Arduino Libraries to ease the coding process.
+
+Line Following Library: [QTR Sensors][def3]
+
+Orientation Library: [Accelerometer][def4] , [Compass][def5]
+
+Position Library: [Grove - Ultrasonic Ranger][def2]
+
+Object Detection Library: [Grove - Time of Flight Distance Sensor VL53L0X][def1] 
+
+[def1]: https://github.com/Seeed-Studio/Grove-Ranging-sensor-VL53L0X
+
+[def2]: https://github.com/SeeedDocument/Grove_Ultrasonic_Ranger
+
+[def3]: https://www.pololu.com/docs/0J19/1
+
+[def4]: https://github.com/adafruit/Adafruit_LSM6DS
+
+[def5]: https://github.com/adafruit/Adafruit_LIS3MDL
+
+### Communication with Jetson Nano
+The Arduino will communicate with the Jetson Nano through the USB port. The USB port will provide serial data to the Jetson Nano using UART (Universal Asynchronous Receiver Transmitter) Communication Protocol. Another option is to use the Tx/Rx pins on the Arduino Mega to send the serial data, rather than the USB. This would allow code to be uploaded to the Arduino Mega while still being serially connected to the Jetson Nano.
+
+By using UART, the serial data must have a beginning marker, such as '<', and an ending marker, such as '>'. This is an effect of the communication method being asynchronous. The start and end markers will allow the Jetson Nano to differentiate the data packages.
+
+### Reach Goal Analysis
```

Comment:
Does ESP 32 come with the bluetooth attribute?

Reply:
The ESP will not be in charge of data transmission form the Arduino to the Jetson Nano. The Arduino/Jetson Nano communication will be done by an internal cord. But to answer your question, yes the ESP has both wifi and bluetooth capabilities.